packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1508,39 +1494,39 @@ abstract class ResidentRunner extends ResidentHandlers {
   bool get reportedDebuggers => _reportedDebuggers;
   bool _reportedDebuggers = false;
 
-  void printDebuggerList({ bool includeVmService = true, bool includeDevtools = true }) {
-    final DevToolsServerAddress? devToolsServerAddress = residentDevtoolsHandler!.activeDevToolsServer;
-    if (!residentDevtoolsHandler!.readyToAnnounce) {
-      includeDevtools = false;
-    }
-    assert(!includeDevtools || devToolsServerAddress != null);
+  void printDebuggerList() {
     for (final FlutterDevice? device in flutterDevices) {
       if (device!.vmService == null) {
         continue;
       }
-      if (includeVmService) {
-        // Caution: This log line is parsed by device lab tests.
-        globals.printStatus(
-          'A Dart VM Service on ${device.device!.name} is available at: '
-          '${device.vmService!.httpAddress}',
-        );
+      // Caution: This log line is parsed by device lab tests.
+      globals.printStatus(
+        'A Dart VM Service on ${device.device!.name} is available at: '
+        '${device.vmService!.httpAddress}',
+      );
+
+      final DartDevelopmentService dds = device.device!.dds;
+      final Uri? dtdUri = dds.dtdUri;
+      if (debuggingOptions.printDtd && dtdUri != null) {
+        globals.printStatus('The Dart Tooling Daemon is available at: $dtdUri');
```

Comment:
can you add back the new line so that the DevTools URI is separated in the CLI output?

Reply:
I removed it so the DevTools URI was grouped with the VM service and DTD messages. I can print a newline after all these messages are printed instead.